cmd/info.go

```diff
@@ -58,7 +60,6 @@ func infoMain(cmd *cobra.Command, args []string) exitCode {
 			}
 
 			entryMap := orderedMap{linkedhashmap.New()}
-			entryMap.Put("Path", entry.Path)
```

Comment:
We can have multiple entries. Path seems like it could still be useful when we printing more than one entry.

Reply:
The output will be a hash of `<path> => <info>`.